test/suites/migration.sh

```diff
@@ -437,6 +437,16 @@ migration() {
   lxc_remote rm -f l1:c1
   lxc_remote rm -f l2:c1
 
+  lxc_remote init testimage l1:c1
```

Comment:
Could you add a comment explaining why we are testing this for future readers. Then this should be good to go assuming the tests pass :)